src/main/java/gov/cabinetoffice/gap/adminbackend/dtos/application/ApplicationSectionOrderPatchDto.java

```diff
@@ -0,0 +1,16 @@
+package gov.cabinetoffice.gap.adminbackend.dtos.application;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+@Builder
+public class ApplicationSectionOrderPatchDto {
+
+    private String sectionId;
+
+    private Boolean upOrDown;
```

Comment:
Probably need to use something different to a boolean here. UpOrDown = true could mean either.

I'd maybe suggest an integer for which index to move the section to instead as this would be a bit more generic too

Reply:
Fixed in latest commit